src/Transactions/TransactionFields/CodeNumberOfficeFields.php

```diff
@@ -64,4 +65,18 @@ public function getNumber(): ?int
     {
         return $this->number;
     }
+
+    /**
+     * Get the booking reference. The booking reference uniquely identifies a booking.
+     *
+     * @return BookingReference
+     */
+    public function getBookingReference(): BookingReference
```

Comment:
I believe that the method on all of the `Line` objects is called `getReference(): MatchReferenceInterface`. So perhaps we should call this `getReference()` as well?